### PR TITLE
FEAT: who doesn't like cookies?

### PR DESCRIPTION
**Pull request by sooox-cc**
- Added a cookie consent notice that appears before any authentication processes.
- Introduced a separate consent cookie to be set prior to any authentication cookies.
- Ensured functionality without JavaScript for broader accessibility.
- Informed users about the use of authentication cookies before they are set.
- Simplified the acceptance process with a straightforward form submission.
- Prevent login and registration actions until users accept cookies, as they are essential for authentication.

### CODE REVIEW

File: public/css/style.css
Comment (SleepingAmi) [inline public/css/style.css:1]: Why did you make a CSS animation neat?

Comment (sooox-cc) [reply public/css/style.css:1]: it happens when i save lol, i have save on format on

Comment (sooox-cc) [reply public/css/style.css:1]: format on save*

**Review by SleepingAmi — APPROVED** [review]
Sure, some changes are a bit OP, but approved

**Pull request merged**